.github/ISSUE_TEMPLATE/new-rule.yml

```diff
@@ -0,0 +1,42 @@
+name: "\U0001F680 Propose a new rule"
+description: "Propose a new rule to be added to the plugin"
+title: "New Rule: (fill in)"
+labels:
+  - rule
+  - feature
+body:
+- type: markdown
+  attributes:
+    value: By opening an issue, you agree to abide by the [Open JS Foundation Code of Conduct](https://eslint.org/conduct).
+- type: input
+  attributes:
+    label: Rule details
+    description: What should the new rule do?
+  validations:
+    required: true
+- type: dropdown
+  attributes:
+    label: What type of rule is this?
+    options:
+      - Warns about a potential problem
+      - Suggests an alternate way of doing something
+      - Enforces a formatting/stylistic preference
```

Comment:
We should probably not offer stylistic rules anymore (I believe they're getting removed?)

Reply:
stylistic rules will be deprecated in eslint v9.

yes, it's fine to remove it - the plugin does not offer stylistic rules.